src/core/libmaven/datastructures/mzSlice.h

```diff
@@ -107,6 +111,8 @@ class mzSlice
             rtmax = b.rtmax;
             ionCount = b.ionCount;
             compound = b.compound;
+            compoundVector = b.compoundVector;
```

Comment:
Why the addition of a compound vector for every slice? When would it have more than one compound assigned to it?

Reply:
As in, where is it utilized. Even if I have x compounds per formula.

Reply:
It is used in `PeakDetector::processSlices` method. Peak-grouping is attempted for each compound in the vector for each slice. Again, as discussed, we may instead do adduct detection like we do it for isotopes.